### PR TITLE
Add night summary script

### PR DESCRIPTION
**Pull request by juanep97**
This PR adds an `iop4-night-summary` command that can be used to build an automatic html summary of the IOP4 results for a given night. The output can be saved to a file and/or sent by email.

The summary is built by querying the DB for the given night. If not night is specified, the default is yesterday (useful for the default use case of generating the summary after the daily run of the pipeline). At the moment, the summary includes an overview of the epochs in the DB and plots of the results.

### CODE REVIEW

File: iop4admin/modeladmins/epoch.py [hunk 1]
```diff
@@ -37,6 +37,10 @@ def count_bias(self, obj):
     def count_flats(self, obj):
         return obj.rawflatcount
     
+    @admin.display(description='Nº of darks')
```
Comment (morcuended) [inline iop4admin/modeladmins/epoch.py:40]: maybe better write Number of darks explicitely

File: iop4admin/modeladmins/epoch.py [hunk 2]
```diff
@@ -37,6 +37,10 @@ def count_bias(self, obj):
...
+    def count_darks(self, obj):
+        return obj.rawdarkcount
+    
     @admin.display(description='Nº of science files')
```
Comment (morcuended) [inline iop4admin/modeladmins/epoch.py:44]: same

**Review by morcuended — DISMISSED** [review]
Looks good, just a very minor comment that can be ignored

**Comment by morcuended** [comment]
Do you want to want for #84?

File: iop4admin/modeladmins/epoch.py (hunk 2 above)
Comment (juanep97) [reply iop4admin/modeladmins/epoch.py:44]: imagen

These are the column names, I use the abbreviation so the table does not get unnecessarily wide (there is already quite a bit of wasted space).

File: iop4admin/modeladmins/epoch.py (hunk 1 above)
Comment (juanep97) [reply iop4admin/modeladmins/epoch.py:40]: (answered in the other comment)

**Comment by juanep97** [comment]
> Do you want to want for #84?

Yep, I think we can wait

File: iop4admin/modeladmins/epoch.py (hunk 2 above)
Comment (morcuended) [reply iop4admin/modeladmins/epoch.py:44]: Alright then

**Review by morcuended — COMMENTED** [review]
you might want to remove the jupyter image committed by accident.

Otherwise the PR looks good and can be merged

**Review by morcuended — APPROVED** [review]

**Comment by morcuended** [comment]
squashing and merging

**Pull request merged**